types/voucher-code-generator/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for voucher-code-generator 1.1
+// Project: http://www.voucherify.io/
+// Definitions by: My Self <https://github.com/me>
```

Comment:
Please add yourself as the definition author

Reply:
Done
`// Definitions by: João Moura <https://github.com/JWebCoder>`